packages/twenty-front/src/pages/settings/data-model/SettingsObjectNewField/SettingsObjectNewFieldStep2.tsx

```diff
@@ -166,6 +165,12 @@ export const SettingsObjectNewFieldStep2 = () => {
                 );
 
                 if (!edges) return viewFieldsRef;
+                const fieldExist = edges.some(
```

Comment:
I don't think this is the right approach. We actually want the field to be added optimistically from settings everytime. The issue is that the field is not added at all.

Then, from the Table view, we can re-add the field (because it has actually not been added in Settings field creation), which result in twice the same field

Reply:
ahh, I got it now! It should be added to the table view as soon as we create a new field. I will try to do this, thanks! 
I will create another commit in the same PR.

Reply:
yes, and the code you have modified should do exactly that but it seems to be failing for some reason. Apollo cache handling is not straight forward, let us know if you need help :)